apps/gallery/js/thumbnails.js

```diff
@@ -21,7 +21,9 @@
   // Bug 1135256: Having a listener in ThumbnailList prevents them from
   // being garbage collected since ready() holds a strong reference, so
   // we have a singleton listener here.
-  navigator.mozL10n.ready(thumbnails.list.localize.bind(thumbnails.list));
+  thumbnails.list.localize();
```

Comment:
l10n is never "not-ready" anymore, that's the nature of asynchronicity of the library now! if you use `document.l10n.formatValue`, it will resolve when resources are loaded. If you set `data-l10n-id`, it will localize when resources are loaded. No need for `ready`/`once` wrappers anymore!